tests/integration/test_max_http_connections_for_replication/test.py

```diff
@@ -68,9 +68,9 @@ def test_keepalive_timeout(start_small_cluster):
 
     assert not node2.contains_in_log("No message received"), "Found 'No message received' in clickhouse-server.log"
 
-node3 = cluster.add_instance('node3', config_dir="configs", main_configs=['configs/remote_servers.xml', 'configs/log_conf.xml'], with_zookeeper=True)
-node4 = cluster.add_instance('node4', config_dir="configs", main_configs=['configs/remote_servers.xml', 'configs/log_conf.xml'], with_zookeeper=True)
-node5 = cluster.add_instance('node5', config_dir="configs", main_configs=['configs/remote_servers.xml', 'configs/log_conf.xml'], with_zookeeper=True)
+node3 = cluster.add_instance('node3', user_configs=[], main_configs=['configs/remote_servers.xml', 'configs/log_conf.xml'], with_zookeeper=True)
```

Comment:
`user_configs=[]` is default value, isn't it?

Reply:
Yes. Leftovers after `sed`.

Reply:
Could you remove it?

Reply:
And `main_configs=[]` as well. Just to make the code shorter.